apps/api/src/main/java/dk/treecreate/api/mail/MailService.java

```diff
@@ -188,6 +190,21 @@ public void sendCustomOrderRequestEmail(CreateCustomOrderRequest orderInfo)
         orderInfo.getImages());
   }
 
+  public void sendOrderPaymentReminderEmail(Order order) throws Exception {
+    String subject = "Your Treecreate order is waiting for your payment!";
+    Context context = new Context(new Locale("da"));
+    context.setVariable("paymentLink", quickpayService.getPaymentLink(order.getPaymentId()).getUrl());
+    System.out.println(quickpayService.getPaymentLink(order.getPaymentId()));
```

Comment:
Don't use normal console logs for this, use the Logger
But this log is redundant anyway since the ordersService already logs this, so it can be removed
```suggestion
```